CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ accidentally triggering the load of a previous DB version.**
 * #5547 Skip Ordered Append when only 1 child node is present
 * #5510 Propagate vacuum/analyze to compressed chunks
 * #5584 Reduce decompression during constraint checking
+* #5530 Improve performance for queries on compressed tables where the query ORDER BY matches the compression ORDER BY
```

Comment:
I think this is a bad PR title and I very much preferred the previous one

Reply:
Maybe we can find a compromise between current one and previous one. I think the current one is way too long. Since we link to the PR people can always click to that and read a more detailed explanation there.

Reply:
There were some conflicting suggestions for the changelog entry/PR title during the review. I have opened a thread in our Slack channel to discuss a good changelog entry.

Reply:
As discussed in the standup, I changed the changelog line to "Optimize compressed chunk resorting"